docs/src/examples/bruss.md

```diff
@@ -32,9 +34,11 @@ kernel_v! = let N = N, xyd = xyd_brusselator, dx = step(xyd_brusselator)
         im1 = limit(i - 1, N)
         jp1 = limit(j + 1, N)
         jm1 = limit(j - 1, N)
-        du[II[i, j, 2]] = α * (u[II[im1, j, 2]] + u[II[ip1, j, 2]] + u[II[i, jp1, 2]] +
-                           u[II[i, jm1, 2]] - 4u[II[i, j, 2]]) +
-                          A * u[II[i, j, 1]] - u[II[i, j, 1]]^2 * u[II[i, j, 2]]
+        du[II[i,
+            j,
```

Comment:
JuliaFormatter v2 is pretty mangled and messes up a lot of lines like this, can we just have the same thing we had with JuliaFormatter v1 @domluna? What's the reason for changing the style?

Reply:
Agreed, this is currently our group's main issue with JuliaFormatter. 
Previously, I just made PRs for things like this myself, but unfortunately, I don't have time to invest into this now.